microsite/data/plugins/env0.yaml

```diff
@@ -5,6 +5,6 @@ authorUrl: https://www.env0.com/
 category: Infrastructure
 description: Use env0 scaffolder actions to deploy and manage your infrastructure using env0 directly from Backstage
 documentation: https://github.com/env0/env0-backstage-plugin
-iconUrl: https://cdn.prod.website-files.com/63df9b273f861215467107c8/668fd303582961e81ef532fc_env0%20logo-dark.svg
+iconUrl: https://avatars.githubusercontent.com/u/46656519?s=200&v=4
```

Comment:
```suggestion
iconUrl: https://avatars.githubusercontent.com/u/46656519
```

Reply:
I see it has no effect, and that is what we got from the team so I prefer to leave it this way